package.json

```diff
@@ -3,5 +3,25 @@
   "private": "true",
   "scripts": {
     "build": "webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "@babel/core": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
+    "@babel/preset-react": "^7.22.15",
+    "@hotwired/stimulus": "^3.2.2",
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update your package.json file by changing the the `build script` and adding the `esbuild` dependency like in the example below. Right now, it happens your app to crash when testing it. But the below code lines fixes the issue:
```json
"scripts": {
    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=/assets --loader:.js=jsx"
  },
  "dependencies": {
    
    .
    .
    .
    "esbuild": "^0.19.5",
    .
    .
    .
  },
```